client/layout/sidebar/footer.jsx

```diff
@@ -3,21 +3,49 @@
  */
 import React from 'react';
 import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import page from 'page';
 
 /**
  * Internal dependencies
  */
-var Gridicon = require( 'components/gridicon' );
-
+import Gridicon from 'components/gridicon';
 import Button from 'components/button';
+import { openChat } from 'state/ui/happychat/actions';
+import viewport from 'lib/viewport';
+import config from 'config';
 
-const SidebarFooter = ( { translate, children } ) => (
+const SidebarFooter = ( { translate, children, onOpenChat } ) => (
```

Comment:
We _do_ expect to always show it at this point.

#7791 removes all of the connect logic from footer and encapsulates it.